.circleci/config.yml

```diff
@@ -94,26 +94,24 @@ commands:
             docker push ${DOCKHUB_ORGANISATION}/binary-static-ticktrade
   k8s_deploy:
     description: "Deploy to k8s cluster"
+    parameters:
+      k8s_namespace:
+        type: string
+        default: "ticktrade-binary-com-production"
+      k8s_service:
+        type: string
+        default: "binary-static-ticktrade"
     steps:
       - k8s/install-kubectl
       - run:
           name: Deploying to k8s cluster for service ticktrade-binary-com
           command: |
-            for SERVER_ID in {1..5}
-            do
-              KUBE_SERVER_REF="KUBE_SERVER_$SERVER_ID"
-              SERVICEACCOUNT_TOKEN_REF="SERVICEACCOUNT_TOKEN_$SERVER_ID"
-              CA_CRT_REF="CA_CRT_$SERVER_ID"
-              if [ ! -z "${!KUBE_SERVER_REF}" ]
-              then
-                echo "Deploying to cluster $SERVER_ID"
-                CA_CRT="${!CA_CRT_REF}"
-                KUBE_SERVER="${!KUBE_SERVER_REF}"
-                SERVICEACCOUNT_TOKEN="${!SERVICEACCOUNT_TOKEN_REF}"
-                echo $CA_CRT | base64 --decode > ca.crt
-                kubectl --server=${KUBE_SERVER} --certificate-authority=ca.crt --token=$SERVICEACCOUNT_TOKEN set image deployment/ticktrade-binary-com ticktrade-binary-com=${DOCKHUB_ORGANISATION}/binary-static-ticktrade:${CIRCLE_TAG}
-              fi
-            done
+            TAG=${CIRCLE_TAG:-$CIRCLE_SHA1}
+            export NAMESPACE=<< parameters.k8s_namespace >>
+            git clone https://github.com/binary-com/devops-ci-scripts
+            cd devops-ci-scripts/k8s-build_tools
+            echo $CA_CRT | base64 --decode > ca.crt
+            ./release.sh << parameters.k8s_service >> ${TAG}
```

Comment:
i think we are missing the image name `binary-static-ticktrade` as the first argument

Reply:
In [this](https://github.com/binary-com/devops-ci-scripts/blob/d94350548869b50fb68222e6103666c9e90141b1/k8s-build_tools/release.sh#L82-L90) script that we use for blue/green deployment, the image name and app name are same `(k8s_service parameter)` . I'm working on a  [robust version](https://github.com/binary-com/devops-ci-scripts/pull/4) for that with more features